sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/healthcare/DicomIO.java

```diff
@@ -51,8 +64,33 @@
  * readMetadataResult.getReadResponse() PCollection<String> failRead =
  * readMetadataResult.getFailedReads() }
  */
+
+// @SuppressWarnings({
+// "nullness" // TODO(https://github.com/apache/beam/issues/20497)
+// })
```

Comment:
I would like to recommend removal of this annotation. My first time trying to pass checks when I did this took a long while. If you would like to work through these together, please let me know. I found that once I learned the tricks of how to deal with them, it was a lot faster and I no longer use this annotation.